d3rlpy/logger.py

```diff
@@ -110,7 +110,7 @@ def add_metric(self, name: str, value: float) -> None:
             self._metrics_buffer[name] = []
         self._metrics_buffer[name].append(value)
 
-    def commit(self, epoch: int, step: int) -> None:
+    def commit(self, epoch: int, step: int) -> dict:
```

Comment:
The dictionary type should be as follows.
```py
def commit(self, epoch: int, step: int) -> Dict[str, float]:
```

Reply:
done !

Reply:
Thanks! I'll merge this when the test has passed.